components-compose/src/main/java/uk/gov/hmrc/components/compose/organism/headline/HeadlineCardView.kt

```diff
@@ -160,15 +160,14 @@ object HeadlineCardView {
                         }
                         .then(
                             if (childPaddingState) {
-                                Modifier.padding(horizontal = HmrcTheme.dimensions.hmrcSpacing16)
+                                Modifier.padding(HmrcTheme.dimensions.hmrcSpacing16)
```

Comment:
Do you need to remove the Spacer above now that you're adding top padding here too?